### PR TITLE
Issue #1107 Updating AppVeyor links

### PR DESCRIPTION
**Pull request by hferentschik**
Addresses issue #1107

### CODE REVIEW

**Comment by budhrg** [comment]
LGTM :+1:

File: README.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ Minishift uses https://github.com/docker/machine/tree/master/libmachine[libmachi
 provisioning VMs, and https://github.com/openshift/origin[OpenShift Origin] for running the cluster.
 
 https://travis-ci.org/minishift/minishift[image:https://secure.travis-ci.org/minishift/minishift.png[Build Status]]
-https://ci.appveyor.com/project/hferentschik/minishift-o61ou/branch/master[image:https://ci.appveyor.com/api/projects/status/6wyv1cpd588cm4ce/branch/master?svg=true[Build status]]
+https://ci.appveyor.com/project/minishift-bot/minishift/branch/master[image:https://ci.appveyor.com/api/projects/status/o0mha7mpanp7dpyo/branch/master?svg=true[Build status]]
```
Comment (praveenkumar) [inline README.adoc]: 👍

File: appveyor.yml [hunk 2]
```diff
@@ -33,3 +33,7 @@ artifacts:
     name: OS-X minishift binary
   - path: out\windows-amd64\minishift.exe
     name: Windows minishift binary
+
+notifications:
+  - provider: Webhook
+    url: http://minibot.34e99f76.svc.dockerapp.io:9009/hubot/appveyor
```
Comment (praveenkumar) [inline appveyor.yml]: Great 👍

Comment (hferentschik) [inline appveyor.yml]: I need to make some more changes on the minot side and re-deploy, but this way the hook is already enabled :-)

**Comment by hferentschik** [comment]
Rebased and merged.

**Pull request closed without merging**